docs/app/sidebars.js

```diff
@@ -30,12 +30,12 @@ const sidebars = {
         }, {
             type: 'doc',
             id: 'quickstart'
-        }, {
-            type: 'doc',
-            id: 'contributor-quickstart'
         }, {
             type: 'doc',
             id: 'devbox_global'
+        },{
+            type: 'doc',
+            id: 'devbox_cloud/index'
```

Comment:
Should this be `devbox_playground`?

Reply:
I could do that, but it would also change the URL structure, which I wasn't sure about